{{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/settings/docker.py

```diff
@@ -8,3 +8,32 @@
 
 # All hosts are allowed to connect to the container by default
 ALLOWED_HOSTS = os.environ.get('DJANGO_ALLOWED_HOSTS', '*').split(',')
+
+# Logging
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format': ('%(levelname)s %(asctime)s %(pathname)s:%(lineno)d '
+                       '%(funcName)s "%(message)s"')
+        },
+        'simple': {
+            'format': '%(levelname)s "%(message)s"'
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'INFO',
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose',
+        },
+    },
+    'loggers': {
+        '': {
+            'handlers': ['console'],
```

Comment:
I'd a 'email' handler for 'ERROR' level logs in production systems as well.

Reply:
OK, we need to port that to the preferences webapp then.